src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -38,8 +38,12 @@ class ContainerAwareEventManager extends EventManager
     /**
      * @param list<string|EventSubscriber|array{string[], string|object}> $subscriberIds List of subscribers, subscriber ids, or [events, listener] tuples
      */
-    public function __construct(ContainerInterface $container, array $subscriberIds = [])
+    public function __construct(ContainerInterface $container, /**@deprecated*/ array $subscriberIds = [])
```

Comment:
Although misnamed, this parameter accepts listeners in addition to subscribers (see phpdoc above and [RegisterEventListenersAndSubscribersPass#L111](https://github.com/symfony/symfony/blob/6.3/src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterEventListenersAndSubscribersPass.php#L111)) so it must not be deprecated.. Only passing subscribers/subscriber ids should be deprecated, which can be done by triggering  the notice from `initializeSubscribers()` (l212) instead.

Reply:
indeed, how could I've missed this phpdoc ! 😅